### ISSUE TITLE
Follow us links missing

### ISSUE THREAD

**Issue by cluhmann** [pymc-devs/pymcon_web_series_website#55]
At the bottom of each page there are a bunch of links.  There is a section all the way to the right with the heading "Follow Us" which I assume is supposed to include a bunch of social media links.  But there is nothing there.
![image](https://user-images.githubusercontent.com/39170995/206611229-d195d894-42ef-48b2-b278-6d9d6c523dd5.png)

**Comment by cluhmann** [comment]
This appears to be a duplicate of a now-closed issue but not entirely resolved issue from the earlier repo: https://github.com/pymc-devs/pymcon_web_series/issues/167

**Comment by cluhmann** [comment]
Yeah, I created the issue before I realized I had already reported it on the old repo.  I don't think it's a major problem.

**Comment by reshamas** [comment]
On the Opera browser, I have also had latency issues with other pages.

However, with the footer, I can see the "Follow Us" links. 

<img width="1221" alt="Screen Shot 2023-04-06 at 10 31 41 AM" src="https://user-images.githubusercontent.com/2507232/230409619-f6c40e22-306f-49e5-ab07-3c6a5ccb6b0a.png">

Closing this issue.  If the problem returns, we can reopen it.
